cdap-ui/app/cdap/components/AbstractWizard/index.js

```diff
@@ -101,7 +101,7 @@ const WizardTypesMap = {
   }
 };
 
-export default function AbstractWizard({isOpen, onClose, wizardType, input, backdrop}) {
+export default function AbstractWizard({isOpen, onClose, wizardType, input, backdrop, usedInMarket, isLastStepInMarket}) {
```

Comment:
I'm not sure if this is the right fix. In order to cater to market wizards we are just modifying a generic component (AbstractWizard) to something specific for a use case. We should still use the abstract wizard and have new wizards for market that handles the calls to action. It might be a bigger change but I feel we should do that instead of modifying a generic component.